schema.js

```diff
@@ -15,6 +16,9 @@ module.exports = Joi.object().keys({
       import: Joi.array().items(Joi.object().keys({
         filename: Joi.string()
       }).requiredKeys('filename').unknown(true)),
+      download: Joi.array().items(Joi.object().keys({
+        sourceURL: Joi.string()
```

Comment:
good idea, @tigerlily-he, but unfortunately I think @orangejulius is right about some folks still relying on older tools.